packages/kbn-es/src/utils/docker.ts

```diff
@@ -514,11 +520,11 @@ export async function setupServerlessVolumes(log: ToolingLog, options: Serverles
 /**
  * Resolve the Serverless ES image based on defaults and CLI options
  */
-function getServerlessImage(options: ServerlessOptions) {
+function getServerlessImage(options: { image?: string; tag?: string }) {
   return resolveDockerImage({
     ...options,
-    repo: SERVERLESS_REPO,
-    defaultImg: SERVERLESS_IMG,
```

Comment:
This is still receiving the full `options` from the parent functions, so I think the type is correct. If we want to be explicit on what is passed into `resolveDockerImage`, we can remove the `...options` and use `options.tag` and `options.image` directly.

Reply:
I strictened these types because they don't need more than these bits from the options. I think it's a fair practise to only ask for what the function needs - wider objects will fit in, but changes to the full options type won't affect the behaviour of these functions anymore. If you think this is confusing, I'm happy to revert these changes, it just helped me understand the whole arg-passing around here :D

Reply:
I don't really think its confusing or needs to be reverted, just should be aligned with what is actually being passed around. If the full `options` object was needed in the future, it is not immediately obvious now.

I'd say remove the spread (it creates ambiguity anyways) and pass in `image` and `tag` only. Then the type is stricter and things are more explicit.

Reply:
That can work too